src/cephadm/cephadm.py

```diff
@@ -7109,6 +7109,9 @@ def command_rm_daemon(ctx):
          verbosity=CallVerbosity.DEBUG)
     call(ctx, ['systemctl', 'disable', unit_name],
          verbosity=CallVerbosity.DEBUG)
+    # force remove any leftover admin socket files
+    call(ctx, ['rm', '-rf', '/var/run/ceph/*.asok'],
+         verbosity=CallVerbosity.DEBUG)
```

Comment:
Please, could you provide more context? 

This change will delete not only the socket for the daemon being removed **but all the asok sockets** but the method `command_rm_daemon` is called each time cephadm tries to remove a daemon. Is this what you are trying to achieve?

Reply:
@alimaredia , to build on what Redo is saying, when we had talked I got the idea you only wanted to remove the asok file for the specific daemon being removed. The current code would remove it from all the daemons (technically it's currently deleting from the wrong dir as @rkachach mentioned so it wouldn't do anything but you know what I mean) rather than just the daemon being removed.

Reply:
If you remove only one, just be careful as librbd might create multiple asoc files.